docs/reference/ml/apis/evaluate-dfanalytics.asciidoc

```diff
@@ -8,15 +8,9 @@
 <titleabbrev>Evaluate {dfanalytics}</titleabbrev>
 ++++
 
-experimental[]
+Evaluates the {dfanalytics} for an annotated index.
```

Comment:
I moved the paragraph down into a "Description" section and drafted a shorter intro for the API here. If there's a better short blurb, that's great.